src/partial-assembly.md

```diff
@@ -0,0 +1,165 @@
+# Performance and Partial Assembly
+
+This document provides a brief overview of the tensor-based high-performance
+and partial assembly features in MFEM. In the traditional finite element
+setting, the operator is assembled in the form of a matrix. The action of the
+operator is computed by multiplying with this matrix. At high orders this
+requires both a large amount of memory to store the matrix, as well as many
+floating point operations to compute and apply it. _Partial assembly_ is a
+technique that allows for efficiently applying the action of finite element
+operators efficiently without forming the corresponding matrix. This is
+particularly important when running on GPUs.
+
+Partial assembly is enabled at the level of the `BilinearForm` by setting
+the assembly level:
+```c++
+a->SetAssemblyLevel(AssemblyLevel::PARTIAL);
+```
+Once partial assembly is enabled, subsequent calls to functions such as
```

Comment:
Technically this is a `method` or `member function` rather than a `function` but perhaps this is overly pedantic.

Reply:
I changed it to `member function`, which I believe is the preferred terminology in C++.